src/go/rpk/pkg/cli/cmd/container/common/common.go

```diff
@@ -204,7 +204,9 @@ func RemoveNetwork(c Client) error {
 func CreateNode(
 	c Client,
 	nodeID, kafkaPort, proxyPort, rpcPort, metricsPort uint,
-	netID, image string,
+	netID string,
+	enableWasm bool,
```

Comment:
why not simply take an array of strings and pass that directly as a `--set "..."`  or simply allow the user to pass `--set` values?

Reply:
I will do that. I will get on that after I fix https://github.com/vectorizedio/redpanda/issues/1411, which should be sometime today or tomorrow.